test/pjrt/test_mesh_service.py

```diff
@@ -1,34 +1,57 @@
-import itertools
-import os
-from typing import List, Optional
+import functools
```

Comment:
is this test being run as part of the CI? I can't seems to find it.

Reply:
This is only good as a sanity check since we can't run XLA collective ops on CPU and we don't support multi-GPU yet, but I just added it anyway.